README.md

```diff
@@ -44,9 +44,9 @@ For a larger example and features demo:
 
 # Uno Platform Features
 * Supported platforms:
-    * Windows 10 (via the standard UWP Toolkit)
+    * Windows 10 and Windows 11 (via the standard UWP and WinUI Toolkit)
```

Comment:
You may also want to update the info here: https://github.com/unoplatform/uno/blob/master/doc/articles/getting-started/requirements.md